synapseclient/__main__.py

```diff
@@ -219,12 +220,22 @@ def copy(args,syn):
     for i in search:
         if i['file.name'] == ent.name:
             raise ValueError('Filename exists in directory you would like to copy to, either rename or check if file has already been copied!')
-    new_ent = synapseclient.File(ent['path'],parent=args.parentid)
+    #CHECK: If the user created the file, copy the file by using fileHandleId else hard copy
+    if profile == ent.properties.createdBy:
```

Comment:
OR use GET /fileHandle/{handleId} and go over to download/upload of the file if you get a 403 error back